Integration tests in master/release branches are failing since account management merged

<!-- **Are you in the right place?**
1. For issues or feature requests, please create an issue in this repository.
2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://slack.rook.io/).
3. Did you already search the existing open issues for anything similar? -->

**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
Since #17041 was merged, the [master/release integration tests](https://github.com/rook/rook/actions/workflows/integration-tests-on-release.yaml) have been failing consistently. 

As seen in [this test](https://github.com/rook/rook/actions/runs/23059536742/job/66982083391), the errors are:
```
+ go test -v -timeout 1800s -run CephHelmSuite github.com/rook/rook/tests/integration
# github.com/rook/rook/pkg/operator/ceph/object
Error: pkg/operator/ceph/object/account.go:27:97: undefined: admin.Account
Error: pkg/operator/ceph/object/account.go:29:16: undefined: admin.Account
Error: pkg/operator/ceph/object/account.go:32:49: adminOpsContext.AdminOpsClient.GetAccount undefined (type *admin.API has no field or method GetAccount)
```

**Expected behavior:**
Passing CI.

**How to reproduce it (minimal and precise):**
<!-- Please let us know any circumstances for reproduction of your bug. -->
See the history for the [Master/Release CI tests](https://github.com/rook/rook/actions/workflows/integration-tests-on-release.yaml).

--- a/.github/workflows/daily-nightly-jobs.yml
+++ b/.github/workflows/daily-nightly-jobs.yml
@@ -12,6 +12,12 @@ defaults:
 permissions:
   contents: read
 
+env:
+  # Required for go-ceph rgw/admin account APIs which are gated behind
+  # the ceph_preview build tag. Can be removed when go-ceph promotes
+  # the account API out of preview.
+  GOFLAGS: "-tags=ceph_preview"
+
 jobs:
   canary-arm64:
     runs-on: [ubuntu-22.04-arm]

--- a/.github/workflows/integration-test-mgr-suite.yaml
+++ b/.github/workflows/integration-test-mgr-suite.yaml
@@ -20,6 +20,12 @@ concurrency:
 permissions:
   contents: read
 
+env:
+  # Required for go-ceph rgw/admin account APIs which are gated behind
+  # the ceph_preview build tag. Can be removed when go-ceph promotes
+  # the account API out of preview.
+  GOFLAGS: "-tags=ceph_preview"
+
 jobs:
   TestCephMgrSuite:
     runs-on: ubuntu-22.04

--- a/.github/workflows/integration-tests-on-release.yaml
+++ b/.github/workflows/integration-tests-on-release.yaml
@@ -15,6 +15,12 @@ defaults:
 permissions:
   contents: read
 
+env:
+  # Required for go-ceph rgw/admin account APIs which are gated behind
+  # the ceph_preview build tag. Can be removed when go-ceph promotes
+  # the account API out of preview.
+  GOFLAGS: "-tags=ceph_preview"
+
 jobs:
   TestCephHelmSuite:
     runs-on: ubuntu-22.04
@@ -213,11 +219,6 @@ jobs:
 
   TestCephObjectSuite:
     runs-on: ubuntu-22.04
-    env:
-      # Required for go-ceph rgw/admin account APIs which are gated behind
-      # the ceph_preview build tag. Can be removed when go-ceph promotes
-      # the account API out of preview.
-      GOFLAGS: "-tags=ceph_preview"
     strategy:
       fail-fast: false
       matrix:

